package.json

```diff
@@ -19,18 +19,18 @@
   "dependencies": {
     "@babel/core": "7.12.10",
     "@babel/preset-env": "7.12.11",
-    "@cfpb/cfpb-atomic-component": "0.7.3",
-    "@cfpb/cfpb-buttons": "0.7.2",
-    "@cfpb/cfpb-core": "0.7.2",
-    "@cfpb/cfpb-expandables": "0.7.3",
-    "@cfpb/cfpb-forms": "0.7.2",
+    "@cfpb/cfpb-atomic-component": "0.7.6",
+    "@cfpb/cfpb-buttons": "0.7.6",
+    "@cfpb/cfpb-core": "0.7.6",
+    "@cfpb/cfpb-expandables": "0.7.6",
+    "@cfpb/cfpb-forms": "0.7.6",
     "@cfpb/cfpb-grid": "0.7.2",
     "@cfpb/cfpb-icons": "0.7.2",
-    "@cfpb/cfpb-layout": "0.7.2",
-    "@cfpb/cfpb-notifications": "0.7.2",
-    "@cfpb/cfpb-pagination": "0.7.2",
-    "@cfpb/cfpb-tables": "0.7.3",
-    "@cfpb/cfpb-typography": "0.7.2",
+    "@cfpb/cfpb-layout": "0.7.6",
+    "@cfpb/cfpb-notifications": "0.7.6",
+    "@cfpb/cfpb-pagination": "0.7.6",
+    "@cfpb/cfpb-tables": "0.7.6",
+    "@cfpb/cfpb-typography": "0.7.6",
```

Comment:
Just want to confirm that these updates are intentional here. Is this bringing in the DS PR you mentioned in the description?

Reply:
Yes. As well as other recent DS updates.